src/main/java/race/Utils/NumberGenerator.java

```diff
@@ -0,0 +1,6 @@
+package race.Utils;
+
+@FunctionalInterface
+public interface NumberGenerator {
+    public int getValue(int num);
```

Comment:
```suggestion
    int getValue(int num);
```
인터페이스에 public은 제거되어도 될 것 같네요 😄